2996_reflection/reflection.md

```diff
@@ -3123,6 +3127,22 @@ An expression convertible to a reflection is said to _represent_ the correspondi
 :::
 :::
 
+### [conv.ptr]{.sref} Pointer conversions {-}
+
+Expand paragraph 2 to disallow conversions to `void *` from pointers of consteval-only types.
+
+::: std
+[2]{.pnum} A prvalue of type "pointer to `$cv$ T`", where `T` is an object type, can be converted to a prvalue of type "pointer to `$cv$ void`". The pointer value ([basic.compound]) is unchanged by this conversion. [If `T` is a consteval-only type ([basic.types.general]), a program that necessitates this conversion is ill-formed.]{.addu}
+
+:::
+
+Expand paragraph 3 to also disallow conversions from pointers of consteval-only type to pointers to base classes that are not of consteval-only type.
```

Comment:
And this would prohibit the _other_ way of doing type erasure, not to mention totally reasonable conversions. Why?